torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8562,12 +8562,12 @@ class ForeachRightmostArgType(enum.Enum):
 
 class ForeachSampleInput(SampleInput):
     ref_args: Any
-    ref_kwargs: Dict[Any, Any]
+    disable_fastpath: Optional[bool]
```

Comment:
Why optional? Wouldn't we want it to be a bool?

Reply:
Removed the optional part and it's a bool now.